solana/src/main/java/org/p2p/solanaj/model/types/Encoding.kt

```diff
@@ -9,6 +9,7 @@ enum class Encoding(
     @SerializedName("base64")
     BASE64("base64"),
 
+    @Deprecated("Slow for solana, use Base 58")
```

Comment:
Maybe you meant `use base 64`? Or possibly this deprecation should be over `BASE64`

Reply:
fixed, thanks!